app/src/lib/editors/win32.ts

```diff
@@ -302,6 +302,17 @@ const editors: WindowsExternalEditor[] = [
     displayNamePrefix: 'PhpStorm',
     publisher: 'JetBrains s.r.o.',
   },
+  {
+    name: 'Android Studio',
+    // registryKeys: registryKeysForJetBrainsIDE('Android Studio')
+    // Android Studio is based on JetBrains IDEs but I am not sure what to put here?
+    registryKeys: [
+      LocalMachineUninstallKey('Android Studio'),
+    ],
+    executableShimPaths: executableShimPathsForJetBrainsIDE('studio'),
```

Comment:
Hey, @detherminal 👋 Nice to see you opening your first PR in this project 🎉 

Here are my suggestions to improve your PR. 
The Android Studio registry entry structure and JetBrains IDEs one are slightly different.
You can't use `executableShimPaths` because you don't have `InstallLocation` in the registry record. ([Link to comment/doc](https://github.com/desktop/desktop/blob/development/app/src/lib/editors/win32.ts#L30)) In the case of Android Studio, you should use `installLocationRegistryKey: 'DisplayIcon'`; that way, the path will be implicitly referred to by the `DisplayIcon` key.

Reply:
Thanks @tsvetilian-ty for chiming in ❤️ 

@detherminal I think those changes @tsvetilian-ty mentions are all you need. Luckily, Google doesn't do crazy stuff with Android Studio versions, so this will be much easier to support :_D